docs/use-viewmodel-components.md

```diff
@@ -14,22 +17,18 @@ export class ViewModel extends ylem.Object {
 export const VMComponent = createViewModelComponent(ViewModel);
 
 const MenuToggle = () => (
-  <div>
-    ...
-    <VMComponent render={ ({ showMenu, toggleMenu }) => (
-      <button onClick={toggleMenu} />
-      { showMenu
-        ? <Menu />
-        : null
-      }
-    )} />
-    ...
-  </div>
+  <VMComponent render={ ({ showMenu, toggleMenu }) => (
```

Comment:
we had added the wrapping div and dots here to show that it will be part of a larger component. an actual implementation where it only rendered the one VMComponent would be fairly pointless.

Reply:
I got that - I put it back, but I figured people are smart enough to use this in a larger component - I was trying to remove distractions from code.